src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+}
+
+body {
+  margin: 0;
+  padding: 0 8px;
+}
+
+.search__bar {
+  margin-top: 20px;
+  display: flex;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search__bar--big {
+  padding: 22px 0;
+}
```

Comment:
move it to search__bar just

Reply:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/55200530/95e42baa-6a44-4711-bffb-72404caaa0e8)